redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/YogaLayout.kt

```diff
@@ -73,11 +73,13 @@ internal class YogaLayout(context: Context) : ViewGroup(context) {
     calculateLayout(
       requestedWidth = when {
         widthMode == MeasureSpec.EXACTLY -> widthSize.toFloat()
+        widthMode == MeasureSpec.UNSPECIFIED -> Size.UNDEFINED
         widthConstraint == Constraint.Fill -> widthSize.toFloat()
         else -> Size.UNDEFINED
       },
       requestedHeight = when {
         heightMode == MeasureSpec.EXACTLY -> heightSize.toFloat()
+        heightMode == MeasureSpec.UNSPECIFIED -> Size.UNDEFINED
```

Comment:
Without the fix this nested spacer is asked to size itself using (mode=AT_MOST, size=0) and it returns a height of 0.